app/views/notes/_comment.html.erb

```diff
@@ -67,17 +81,16 @@
         </a>
       <% end %>
       <% if logged_in_as(['admin', 'moderator']) || (current_user && (comment.uid == current_user.uid || comment.parent.uid == current_user.uid)) %>
-        <a aria-label="Delete comment" rel="tooltip" title="Delete comment" class="btn btn-outline-secondary btn-sm" id="c<%= comment.cid %>delete-btn">
+        <a aria-label="Delete Comment" rel="tooltip" title="Delete Comment" class="btn btn-outline-secondary btn-sm" id="c<%= comment.cid %>delete-btn">
           <i class='icon fa fa-trash'></i>
         </a>
       <% end %>
       <% if current_user %>
-        <button style="border: 1px solid;padding: 4px 6px;" class="btn btn-outline-secondary dropdown" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-          <i class='icon fa fa-smile-o'></i>
-          <sup style="font-size: 12px;">+</sup>
-        </button>    
+        <a aria-label="Leave an Emoji Reaction" rel="tooltip" title="Leave an Emoji Reaction" class="btn btn-outline-secondary btn-sm dropdown" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          <i class='far fa-heart'></i>
+        </a>
```

Comment:
I changed the icon for emoji reactions:

**Before:** (button slightly bigger than the other buttons, two icons squashed inside button, smiley face)
<img width="129" alt="Screen Shot 2021-01-25 at 9 22 36 PM" src="https://user-images.githubusercontent.com/4361605/105803965-754f7d00-5f53-11eb-9aeb-d20bda8fc594.png">

**After:** (button same size as other buttons, single heart icon)
<img width="170" alt="Screen Shot 2021-01-25 at 9 20 06 PM" src="https://user-images.githubusercontent.com/4361605/105803991-84362f80-5f53-11eb-999a-64134dbfd759.png">